### PR TITLE
add Django 2.1 and Python 3.7 builds

### PR DESCRIPTION
**Pull request by rjw57**
This PR adds configuration for Python 3.7 and Django 2.1 images.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -43,6 +43,14 @@ jobs:
       - TAGS=3.6-alpine alpine latest
       - PYTHON_VERSION=3.6
       - REQUIREMENTS=base
+  python-3.7-alpine:
+    <<: *build_and_push
+    environment:
+      - IMAGE_TYPE=python
+      - IMAGE_NAME=python
+      - TAGS=3.6-alpine alpine latest
```
Comment (abrahammartin) [inline .circleci/config.yml]: tags are wrong

File: .circleci/config.yml [hunk 2]
```diff
@@ -51,6 +59,30 @@ jobs:
...
+      - TAGS=2.1 2.1-py3.6 2.1-py3.6-alpine py3.6 py3.6-alpine 2.1-alpine alpine latest
+      - PYTHON_VERSION=3.6
+      - REQUIREMENTS=django-2.1
+  django-2.1-python-3.7-alpine:
+    <<: *build_and_push
+    environment:
+      - IMAGE_TYPE=python
+      - IMAGE_NAME=django
+      - TAGS=2.1 2.1-py3.6 2.1-py3.6-alpine py3.6 py3.6-alpine 2.1-alpine alpine latest
+      - PYTHON_VERSION=3.6
```
Comment (abrahammartin) [inline .circleci/config.yml]: wrong version of python

File: .circleci/config.yml [hunk 3]
```diff
@@ -51,6 +59,30 @@ jobs:
...
+      - REQUIREMENTS=django-2.1
+  django-2.1-python-3.7-alpine:
+    <<: *build_and_push
+    environment:
+      - IMAGE_TYPE=python
+      - IMAGE_NAME=django
+      - TAGS=2.1 2.1-py3.6 2.1-py3.6-alpine py3.6 py3.6-alpine 2.1-alpine alpine latest
```
Comment (abrahammartin) [inline .circleci/config.yml]: wrong tags

**Review by abrahammartin — CHANGES_REQUESTED** [review]

**Comment by rjw57** [comment]
Tags fixed now :S.

File: .circleci/config.yml [hunk 4]
```diff
@@ -51,6 +59,30 @@ jobs:
       - TAGS=2.0 2.0-py3.6 2.0-py3.6-alpine py3.6 py3.6-alpine 2.0-alpine alpine latest
       - PYTHON_VERSION=3.6
       - REQUIREMENTS=django-2.0
+  django-2.0-python-3.7-alpine:
+    <<: *build_and_push
+    environment:
+      - IMAGE_TYPE=python
+      - IMAGE_NAME=django
+      - TAGS=2.0 2.0-py3.7 2.0-py3.7-alpine py3.7 py3.7-alpine 2.0-alpine alpine latest
```
Comment (abrahammartin) [inline .circleci/config.yml]: shouldn't have alpine or latest as django-2.1-python-3.7-alpine does

File: .circleci/config.yml (hunk 2 above)
Comment (abrahammartin) [reply .circleci/config.yml]: still wrong version of python

File: .circleci/config.yml [hunk 5]
```diff
@@ -51,6 +59,30 @@ jobs:
...
+      - PYTHON_VERSION=3.7
+      - REQUIREMENTS=django-2.0
+  django-2.1-python-3.6-alpine:
+    <<: *build_and_push
+    environment:
+      - IMAGE_TYPE=python
+      - IMAGE_NAME=django
+      - TAGS=2.1 2.1-py3.6 2.1-py3.6-alpine py3.6 py3.6-alpine 2.1-alpine alpine latest
```
Comment (abrahammartin) [inline .circleci/config.yml]: shouldn't have alpine or latest as django-2.1-python-3.7-alpine does

**Review by abrahammartin — CHANGES_REQUESTED** [review]

**Review by abrahammartin — APPROVED** [review]

**Pull request merged**